### DISCUSSION TITLE
Discord DMs

### DISCUSSION THREAD

**Discussion by HellFire123451** [victornpb/undiscord#597] [1. Questions & Answers]
I am trying to figure out how to deleted discord dms messages but it's not working how can I delete them? @victornpb it just says server id/channel id! where is dm stuff?

**Comment by viiper2000** [comment]
The function also works with DMs in pretty much the same manner as it does with servers. Open a DM with a person, open Undiscord, use the CURRENT button to autofill Author ID, Server ID and the Channel ID. (Order typically doesn't make a difference in my experience, but I do go in order from top down.)

From there, fine tune the filters, intervals and advanced settings to your liking and then click the big red button to start. You'll get a prompt in your browser, accept it and it'll start the automated deleting process.

Hope I helped.
